ironfish/src/rpc/routes/chain/getTransaction.test.ts

```diff
@@ -19,9 +19,14 @@ describe('Route chain/getTransaction', () => {
     const transaction = block2.transactions[0]
 
     const notesEncrypted: string[] = []
+    const notes: { hash: string; serialized: string }[] = []
 
     for (const note of transaction.notes) {
       notesEncrypted.push(note.serialize().toString('hex'))
```

Comment:
Is there some way we can mark this for deprecation I wonder?

Reply:
I marked this as deprecated in our docs. I can add a JsDoc as well

Reply:
https://github.com/iron-fish/ironfish/pull/3850